exercise_perseus_render/assets/src/vue/perseus.vue

```diff
@@ -38,6 +37,8 @@
 
 <script>
 
+  const actions = require('kolibri/coreVue/vuex/actions');
```

Comment:
I just merged in this PR - https://github.com/learningequality/kolibri/pull/589

so this line will change to:

```javascript
const actions = require('kolibri.coreVue.vuex.actions');
```